gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/transports/rest.py.j2

```diff
@@ -269,7 +269,7 @@ class {{service.name}}RestTransport({{service.name}}Transport):
 
         {% if not method.client_streaming %}
         {% if method.input.required_fields %}
-        __REQUIRED_FIELDS_DEFAULT_VALUES: Dict[str, str] =  {
+        __REQUIRED_FIELDS_DEFAULT_VALUES: Dict = {
```

Comment:
But only checking for `Dict` does even less type checking than `Dict[str, Any]`, right, so what's the point? I was wondering whether we could have the `str` check on the key but also make the type check tighter than just an `Any`. Does that make sense.

The more I think about it, the more I wonder whether it's a sensible question. I guess each required field can be of any proto-derived type, and there may be multiple required fields, so at best we could check for the union of the types of the required fields in the request message. Hmmmm.... Maybe you're right, `Dict[str,Any]` is the most sensible. Thoughts?

Reply:
Thought about it some more. I suggest reverting your last commit and going back to `Dict[str,Any]` (sorry!) unless there's any way to make the `Any` more specific, which I'm thinking there isn't without a lot more work for arguably small benefit.....

Reply:
Done